pkg/restapi/operation/operations_test.go

```diff
@@ -31,7 +32,8 @@ import (
 )
 
 const (
-	testVaultID = "urn:uuid:abc5a436-21f9-4b4c-857d-1f5569b2600d"
+	testReferenceID = "testReferenceID"
+	testVaultID     = "a55d54b2-fd1b-11ea-adc1-0242ac120002"
```

Comment:
This isn't quite a VaultID... since a vault ID is base58-encoded. By itself, this is really just a testUUID.

Reply:
done